connectors/sources/sharepoint_online.py

```diff
@@ -1174,97 +1194,121 @@ async def get_access_control(self):
-                                )
-                                if owner_access_control_doc:
-                                    yield owner_access_control_doc
-
-                    elif is_person(user):
-                        email = user.get("EMail", user.get("mail", None))
-                        username = user.get(
-                            "UserName", user.get("userPrincipalName", None)
-                        )
-
-                        if _already_seen(email, username):
-                            continue
-
-                        update_already_seen(email, username)
-
-                        person_access_control_doc = await self._user_access_control_doc(
-                            user
-                        )
-                        if person_access_control_doc:
-                            yield person_access_control_doc
+                                update_already_seen(domain_group_id)
+                                async for member_email, member_username in _emails_and_usernames_of_domain_group(
+                                    domain_group_id,
+                                    self.client.group_members,
+                                ):
+                                    if _already_seen(member_email, member_username):
+                                        continue
+
+                                    update_already_seen(member_email, member_username)
+
+                                    member = await self.client.user(member_username)
+                                    member_access_control_doc = (
+                                        await self._user_access_control_doc(member)
+                                    )
+                                    if member_access_control_doc:
+                                        yield member_access_control_doc
+
+                                async for owner_email, owner_username in _emails_and_usernames_of_domain_group(
+                                    domain_group_id,
+                                    self.client.group_owners,
+                                ):
+                                    if _already_seen(owner_email, owner_username):
+                                        continue
+
+                                    update_already_seen(owner_email, owner_username)
+
+                                    owner = await self.client.user(owner_username)
+                                    owner_access_control_doc = (
+                                        await self._user_access_control_doc(owner)
+                                    )
+                                    if owner_access_control_doc:
+                                        yield owner_access_control_doc
+
+                        elif is_person(user):
+                            user_doc = await process_user(user)
+                            if not user_doc:
+                                continue
+                            yield user_doc
```

Comment:
```suggestion
                            if user_doc:
                                yield user_doc
```

nit: just to make it consistent with the other places